quinn-proto/src/endpoint.rs

```diff
@@ -66,10 +66,11 @@ impl Endpoint {
         config: Arc<EndpointConfig>,
         server_config: Option<Arc<ServerConfig>>,
         allow_mtud: bool,
-        rng_seed: Option<[u8; 32]>,
```

Comment:
This is a semver-incompatible change. Not sure we want to merge this as is now?

Reply:
We could retain semver compat by letting this override the endpoint config.

Reply:
I don't really have a preference, so whatever works for you ;)

Shall I modify the code to apply @Ralith 's suggestion? Or did you have a different idea in mind? I'm also ok waiting till enough breaking changes accumulate to roll this PR up with the rest (I can keep using my fork in the meantime).

Reply:
Yeah, that idea works for me! We should potentially file a "breaking change" issue for ourselves to consider when we go to release 0.12.

Reply:
Just reverted the breaking change and updated the documentation. Let me know if there's anything else to be done